packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -1100,6 +1100,28 @@ const ArrayMixin = Mixin.create(Enumerable, {
     argument for any item in the array. This method is often simpler/faster
     than using a callback.
 
+    ```javascript
+    class Jedi {
```

Comment:
Are native classes preferred over plain objects in the docs?
https://github.com/emberjs/ember.js/pull/18505/files

Reply:
@mcfiredrill I don't think so, but I know that native classes are preferred over ES6 modules. Also, I didn't notice your PR. I will close this. Thanks!

Reply:
Thanks, I'm ok to merge whichever one is best. :)